core/trino-main/src/main/java/io/trino/metadata/CatalogMetadata.java

```diff
@@ -167,7 +167,7 @@ private static void safeAbort(CatalogTransaction transaction)
             transaction.abort();
         }
         catch (Exception e) {
-            log.error(e, "Connector threw exception on abort");
+            log.debug(e, "Connector threw exception on abort");
```

Comment:
isn't it an erroneous situation that should be fixed?
if it's not reported, how do we know about this

Reply:
See above: https://github.com/trinodb/trino/pull/19554#issuecomment-1790828066